### ISSUE TITLE
faster rcnn labels:have annotation and not have annotation image fit in faster_rcnn model, loss calculate

### ISSUE THREAD

**Issue by QianLingjun** [facebookresearch/detectron2#2027]
In train faster rcnn model, I have annotation and not have annotation image fit in faster_rcnn model, not have annotation image will participate in the calculation of loss, and will participate in model optimization? I find no ground truth proposal will be labeled -1, mean this proposal not participate loss calculate,. Thinks

**Comment by ppwwyyxx** [comment]
If an image has no annotations, it means the image is annotated to have no objects and it will be trained so the model will predict no objects on such images.
This can be controlled by `cfg.DATALOADER.FILTER_EMPTY_ANNOTATIONS`.

**Comment by QianLingjun** [comment]
Tanks for your reply, I have done this question, and print one image of annotated and not&nbsp;annotated loss, find&nbsp;annotated image return loss is large(0.3),and not annotated image return loss is small (0.0003).&nbsp;Can you give me the explanation of this loss? this loss is print in datectron2/engin/train_loop.py.
Thinks again.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "facebookresearch/detectron2"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年9月13日(星期天) 凌晨3:21
收件人:&nbsp;"facebookresearch/detectron2"<detectron2@noreply.github.com&gt;;
抄送:&nbsp;"钱令军"<1763469890@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [facebookresearch/detectron2] faster rcnn labels:have annotation and not have annotation image fit in faster_rcnn model, loss calculate (#2027)





 
If an image has no annotations, it means the image is annotated to have no objects and it will be trained so the model will predict no objects on such images.
 This can be controlled by cfg.DATALOADER.FILTER_EMPTY_ANNOTATIONS.
